### ISSUE TITLE
Error creating thread: json: cannot unmarshal object into Go value of type bool

### ISSUE THREAD

**Issue by chrismwendt** [sourcegraph/sourcegraph-code-discussions#7]
![2018-10-16 13 22 50](https://user-images.githubusercontent.com/1387653/47045021-9e62ef80-d146-11e8-8abc-100c952de9ae.gif)

Here's how I encountered this:

- Visit https://sourcegraph.sgdev.org/github.com/sourcegraph/website/-/blob/STYLEGUIDE.md?view=code#L15%3A2=&tab=discussions&createThread=true

The discussions extension is not disabled in my user settings or client settings. Is anyone else able to repro? If not, what might be different between our accounts?

cc @slimsag

**Comment by slimsag** [comment]
Sorry you are running into this still, I forgot to track this down. I will figure out what is going on here tonight. I think this _has_ to be something to do with your account settings. You've tried in an incognito window already, so it can't be client settings, correct?

**Comment by chrismwendt** [comment]
Yeah, I just tried incognito, same error. Here are my user settings:

```json
{
  "extensions": {
    "langserver/java": {},
    "sourcegraph/codecov": true,
    "sourcegraph.sgdev.org/sourcegraph/cx-basic-code-intel": false,
    "sourcegraph.sgdev.org/chris/test": true,
    "sourcegraph.sgdev.org/chris/hello-world": true,
    "langserver/javascript": false,
    "langserver/typescript": false,
    "langserver/go": true,
    "sqs/sourcegraph-hello-world": false,
    "sourcegraph.sgdev.org/chris/cx-hello-world-hover": true
  },
  "codecov.decorations": {
    "hide": true
  },
  "codecov.decorations.lineCoverage": false
}
```
